include/openthread/ip6.h

```diff
@@ -464,6 +468,16 @@ otError otIp6AddUnsecurePort(otInstance *aInstance, uint16_t aPort);
  */
 otError otIp6RemoveUnsecurePort(otInstance *aInstance, uint16_t aPort);
 
+/**
+ * This function removes all ports from the allowed unsecure port list.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ *
+ * @retval OT_ERROR_NONE  All ports are successfully removed from the allowed unsecure port list.
+ *
+ */
+otError otIp6RemoveAllUnsecurePorts(otInstance *aInstance);
```

Comment:
I don't think this operation can fail. 
How about changing it to `void` and not returning an `otError`?

Reply:
Updated.